dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/DataManagerTest.kt

```diff
@@ -200,14 +200,14 @@ class DataManagerTest(
 
         `when`(
             mockCloudEventMessageHandler.buildCEMessageAndSendToQueue(
-                dataUUId, MessageType.DataReceived, correlationId, ExchangeNames.dataReceived,
+                anyString(), anyString(), anyString(), anyString(), anyString(),
```

Comment:
The way I understand this test, it doesn't matter what inputs are set here, because the method shall always throw an `AmqpException` in the test.

Is that the reason why you changed this?

Reply:
Ihr könnt entscheiden, ob wir den Test behalten wollen.

Ich akzeptiere für mich einfach, dass hier `anyString()` geschrieben wurde, um zu zeigen, dass die Einträge völlig egal sind an dieser Stelle.

Reply:
anyString is gone again. The test will stay because it prevents changing the code such that no message is sent